project/style.css

```diff
@@ -0,0 +1,227 @@
+.login-section{
+  margin-left: 15%;
+  margin-right:15%;
+  margin-top: 35%;
+}
+
+.section-image1{
+  width:30%;
+  float:left;
+}
+
+.section-image2{
+  height:25%;
+  width:25%;
+  float:right;
+  margin-left: 52%;
+  position:absolute;
+}
+
+.right-paragraph {
+  margin-left: 25%;
+  float: right;
+  position:absolute;
+  width: 40%;
+}
+
+.left-paragraph {
+  float: left;
+  position: absolute;
+  margin-right: 25%;
+  width: 40%;
+}
+
+.header-form{
+  background-color: rgb(51, 153, 255);
+  color: rgb(255, 255, 255);
+  box-shadow: 0px 10px 30px rgb(166, 166, 166);
+  height: 20%;
+  margin-top: 0%;
+  text-align: center;
+}
+
+br { font-size: 200%; }
```

Comment:
Well, this is very, very interesting! I see what you're doing here, but I don't agree with it. Page layout should not be enforced through markup. Rather, the markup should dictate _what_ data to render, while CSS should be the authority on _how_ to render that data. In most cases, if you feel the need to use `<br>`, there are probably better alternatives, such as `margin` / `padding`.